### PR TITLE
adding accept page

### PR DESCRIPTION
**Pull request by johnkuney**
adding accept-bitcoin-cash page

### CODE REVIEW

**Review by deadalnix — CHANGES_REQUESTED** [review]
A small change, but overall, LGTM

File: v2/src/assets/icons/accept/calculator.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 512 512'><title>ionicons-v5-d</title><rect x='112' y='48' width='288' height='416' rx='32' ry='32' style='fill:none;stroke-linecap:round;stroke-linejoin:round;stroke-width:32px'/><rect x='160.01' y='112' width='191.99' height='64' style='fill:none;stroke-linecap:round;stroke-linejoin:round;stroke-width:32px'/><circle cx='168' cy='248' r='24'/><circle cx='256' cy='248' r='24'/><circle cx='344' cy='248' r='24'/><circle cx='168' cy='328' r='24'/><circle cx='256' cy='328' r='24'/><circle cx='168' cy='408' r='24'/><circle cx='256' cy='408' r='24'/><rect x='320' y='304' width='48' height='128' rx='24' ry='24'/></svg>
```
[the remark this answers is not in the document]
Comment (johnkuney) [reply v2/src/assets/icons/accept/calculator.svg]: Okay moved them. Please merge the Spend page PR first though due to cart icon repetition

File: v2/src/pages/accept-bitcoin-cash.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
+import React from "react"
+import fbt from "fbt"
+import SEO from "components/seo"
+import { Container, Row } from "react-bootstrap"
+import Sidebar from "components/sidebar/sidebar.js"
+import LargeTile from "components/large-tile/large-tile.js"
+import Cart from "assets/icons/cart.svg"
```
Comment (deadalnix) [inline v2/src/pages/accept-bitcoin-cash.js:7]: This file is missing.

**Review by deadalnix — DISMISSED** [review]
I wish I could merge this, but the cart is missing.

File: v2/src/pages/accept-bitcoin-cash.js (hunk 2 above)
Comment (johnkuney) [reply v2/src/pages/accept-bitcoin-cash.js:7]: whoops okay its in there now

**Pull request merged**